tests/cugraph/test_basics.py

```diff
@@ -1,10 +1,11 @@
+import cugraph
```

Comment:
Could you leave a comment with a link to the bug tracker so we know where to look up the status?

Reply:
```suggestion
# NOTE(vibwu): Currently cugraph must be imported before torch to avoid a resource cleanup issue.
#    See https://github.com/rapidsai/cugraph/issues/2718
import cugraph
```